pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/KitRequest.java

```diff
@@ -42,7 +42,7 @@ public KitRequest(String participantId, String shortId, DDPParticipant participa
         this(null, participantId, shortId, null, externalOrderNumber, participant, null, null, null);
     }
 
-    public KitRequest(Long dsmKitRequestId, String participantId, String shortId, String ddpLabel, String externalOrderNumber,
+    public KitRequest(Integer dsmKitRequestId, String participantId, String shortId, String ddpLabel, String externalOrderNumber,
```

Comment:
Is it normal for there to be a kit request without a `dsmKitRequestId`?  If not, this should be an `int` to avoid creating an object with invalid/unexpected state.